nuxeo-apidoc-webengine/src/main/resources/skin/views/operation/default.ftl

```diff
@@ -17,6 +17,12 @@
     </div>
   </#if>
 
+  <div class="description">
+    <h2>Description</h2>
+    ${nxItem.description}
+    <@viewSecDescriptions docsByCat=docs.getDocumentationItems(Context.getCoreSession()) title=false/>
```

Comment:
Not not needed, the content is in markdown and already sanitized: see https://github.com/nuxeo/nuxeo-apidoc-server/blob/master/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/documentation/ResourceDocumentationItem.java#L91